### ISSUE TITLE
Missing in Sphinx documentation

### ISSUE THREAD

**Issue by nsoranzo** [codejamninja/sphinx-markdown-parser#3]
Should this be added to https://www.sphinx-doc.org/en/master/usage/markdown.html as an alternative to recommonmark ?

**Comment by clayrisser** [comment]
Yeah, probably so. It's definitely a more feature rich alternative.
